src/Tribe/Tickets_View.php

```diff
@@ -170,6 +185,14 @@ public function update_tickets() {
 		$post_id = get_the_ID();
 		$attendees = ! empty( $_POST['attendee'] ) ? $_POST['attendee'] : array();
 
+		/**
+		 * Sort list to handle all not attending first
+		 *
+		 * switch to the following once WordPress 4.7 is minimum supported:
+		 * $attendees = wp_list_sort( $attendees, 'order_status', 'ASC', true );
```

Comment:
I would check if `wp_list_sort` exists, use it, otherwise fall back to what you did, and add a `@todo` for that.